src/AutoRest.CSharp.V3/ClientModels/SerializationFormat.cs

```diff
@@ -6,9 +6,10 @@ namespace AutoRest.CSharp.V3.ClientModels
     internal enum SerializationFormat
     {
         Default,
-        DateTimeRFC1123,
-        DateTimeISO8601,
-        DateTimeUnix,
-        Date
+        DateTime_RFC1123,
+        DateTime_ISO8601,
+        DateTime_Unix,
+        Date_ISO8601,
+        Duration_ISO8601
```

Comment:
I don't know. But, just like DateTime, there could be. We don't really have a clear format list beyond what is on the test server. And, I haven't done the date/datetime/duration specific sets of tests yet. But, I wouldn't take the test server to be gospel. There is a 'thing' here, like a pattern. And, I don't know if this is the right pattern. But, I'd rather have the implementation follow this pattern for now, and we can split it up or doing a different pattern later. Most of our progress going forward should be like this, and we should look for patterns to refactor when it seems like a good time to refactor them. Right now, I'd rather not reinvent the wheel for every change to allow tests to run. Let's get tests working and we can use those as a good baseline to do refactoring later.

Reply:
Basically, "doing it right the first time" shouldn't be our goal right now. But, recognizing that things can be better, taking note of them (as an issue or as a TODO or otherwise) as we're closing the testserver gap, should be the working strategy.